board/drivers/can.h

```diff
@@ -55,6 +61,7 @@ can_buffer(tx1_q, 0x100)
 can_buffer(tx2_q, 0x100)
 can_buffer(tx3_q, 0x100)
 can_buffer(txgmlan_q, 0x100)
+// cppcheck-suppress misra-c2012-9.3
```

Comment:
Probably fixed by doing `can_ring *can_queues[4] = {&can_tx1_q, &can_tx2_q, &can_tx3_q, &can_txgmlan_q};`

Reply:
Nope, that's what I checked first